app/controllers/idv/in_person/address_search_controller.rb

```diff
@@ -35,6 +41,17 @@ def addresses(search_term)
           response_status_code: err.respond_to?(:response_status) && err.response_status,
         )
         { json: [], status: :unprocessable_entity }
+      rescue StandardError => err
```

Comment:
or even better, since this is the only place the `idv_arcgis_request_failure` event is logged, you could merge it into `id_in_person_location_searched` and make one event that captures everything we need to know about both.

Reply:
i did wonder if it would make sense to do that- but we do also have a similar event for errors in the `usps_locations_controller`. Keeping this here would be in line with what's happening in the other file. Or perhaps refactor both files to log all errors in the search event added in this pr?